sdk/keyvault/Microsoft.Azure.KeyVault/tests/SampleSnippets.cs

```diff
@@ -363,10 +377,16 @@ private async Task SecretsMigrationGuide()
             #region Snippet:Microsoft_Azure_KeyVault_Secrets_Snippets_MigrationGuide_CreateWithOptions
             using (HttpClient httpClient = new HttpClient())
             {
-                //@@AzureServiceTokenProvider provider = new AzureServiceTokenProvider();
-                /*@@*/ provider = new AzureServiceTokenProvider();
-                //@@KeyVaultClient client = new KeyVaultClient(
-                /*@@*/ client = new KeyVaultClient(
+#if SNIPPET
+                AzureServiceTokenProvider provider = new AzureServiceTokenProvider();
+#else
+                provider = new AzureServiceTokenProvider();
+#endif
```

Comment:
Worth merging?

Reply:
Yeah. I did that in some of the latter files but forgot to go back and fix a few of these.